chainerx_cc/chainerx/routines/misc.h

```diff
@@ -25,4 +25,9 @@ Array Minimum(const Array& x1, Scalar x2);
 Array Minimum(Scalar x1, const Array& x2);
 Array Minimum(const Array& x1, const Array& x2);
 
+Array Clip(const Array& x1, Scalar min, Scalar max);
+Array Clip(const Array& x1, const Array& a_min, Scalar max);
+Array Clip(const Array& x1, Scalar min, const Array& a_max);
+Array Clip(const Array& x1, const Array& min, const Array& max);
```

Comment:
How about using the same variables names in NumPy?

Reply:
`a_` prefix seems to mean it may take array object so I think it's shouldn't be used in language that supports function overloading.